### PR TITLE
[DO NOT MERGE, Translation zh-tw] Resources > Translations.md

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/378)

### CODE REVIEW

File: src/content/zh-tw/resources/translations.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+3. 尋找您想要翻譯的文章。
+4. 開始翻譯。
+5. 當翻譯完畢，放送您的翻譯以進行檢查。
+6. 檢查完畢的文章將會被語言管理員作為Pull Request發送到GitHub。
+
+您可以從GitLocalize的[幫助頁面](https://docs.gitlocalize.com/)學到GitLocalize是如何操作的。
+
+如果您遇到任何問題或者有功能請求，您可以在[GitLocalize的issue tracker](https://github.com/gitlocalize/feedback/issues)提供建議。
+
+與其他翻譯貢獻者聊天，您可以[註冊ChromiumDev的Slack](https://chromiumdev-slack.herokuapp.com/)並加入#l10n頻道。
```
Comment (petele) [inline src/content/zh-tw/resources/translations.md]: Need to keep the `Note:` styles, as its a special case for our markdown processor.

File: src/content/zh-tw/resources/translations.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+這是目前所支持的語言：阿拉伯文 (AR), 德文 (DE), 西班牙文 (ES),
+法文 (FR), 希伯來文 (HE), 印尼文 (ID), 義大利文 (IT), 日文 (JA),
+韓文 (KO), 荷蘭文 (NL), 波蘭文 (PL), 葡萄牙文 (PT-BR), 俄文 (RU),
+土耳其文 (TR), 繁體中文 (ZH-TW) 和 簡體中文 (ZH-CN).
+
+## 功勞
+
+我們希望您能得到翻譯文章的功勞。
+
+將您的資料添加到`src/data/_contributors.yaml`，並將`- translator` 添加到 `role` 屬性。 我們使用這資料以讓您的資料出現在[貢獻者頁面](/web/resources/contributors)以及您翻譯的文章。 例如：
```
Comment (petele) [inline src/content/zh-tw/resources/translations.md]: Please break lines at [80 chars](https://developers.google.com/style/html-formatting#line-length)

File: src/content/zh-tw/resources/translations.md (hunk 1 above)
Comment (petele) [reply src/content/zh-tw/resources/translations.md]: I don't see the note bit?

**Comment by petele** [comment]
This still feels like it's missing a lot of content!

**Comment by petele** [comment]
@limhenry - can you take a peek at this one, it seems to be missing content.

**Comment by limhenry** [comment]
See #5607

**Pull request closed without merging**